src/app/hm-admin/content/content.component.scss

```diff
@@ -2,28 +2,52 @@
 @import "../../../core/style/theme-functions";
 
 md-sidenav-layout[fullscreen] {
-  top: 58px;
+  top: 50px;
+
+  md-sidenav {
+    min-width: 200px;
+    max-width: 250px;
+  }
 
-  md-sidenav { min-width: 200px; max-width: 250px; }
 
   md-list-item {
-    display: block; padding: 0; height: 40px;
+    display: block;
+    padding: 0;
     text-align: left;
 
-    h3, p { margin: 0; }
+    h3, p {
+      margin: 0;
+    }
     p {
-      transform: translate3d(0, -200%, 0); opacity: 0; transition: all 200ms linear;
-      font-size: rem(1.2); line-height: 1; color: md-color($md-foreground, secondary-text);
+      transform: translate3d(0, -100%, 0);
+      opacity: 0;
+      transition: all 200ms linear;
+      font-size: rem(1.2);
+      line-height: 1;
+      text-transform: initial;
+      color: md-color($md-foreground, secondary-text);
     }
 
     &.router-link-active,
     &:hover {
       background-color: transparent;
       color: md-color($md-primary, 300);
 
-      p { opacity: 1; transform: translate3d(0, -80%, 0); }
+      p {
+        opacity: 1;
+        transform: translate3d(0, -30%, 0);
+      }
+    }
+    &.router-link-active h3 {
+      font-weight: bold;
     }
-    &.router-link-active h3 { font-weight: bold; }
 
   }
 }
+
+.app-content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 800px;
```

Comment:
to infinity and beyooooond !!!!

Jokes aside, I may miss something, but why not the contrary?

```
width: 95%;
max-width: 800px;
```

Since, the max you don’t want to go over is in fact 800px, not 95%.